cmake/FindCybSDK.cmake

```diff
@@ -1,86 +1,67 @@
-# Copyright 2018 Istituto Italiano di Tecnologia (IIT)
-# @author Mohamed Babiker Mohamed Elobaid <mohamed.elobaid@iit.it>
+# Copyright 2020 Istituto Italiano di Tecnologia (IIT)
+# @author Kourosh Darvish <kourosh.darvish@iit.it>
 
-# Finds the Cyb SDK
+# Finds the Sense Glove SDK
 #
 # This will define the following variables::
 #
-#   CybSDK_FOUND    - True if the system has the CybSDK 
-#   CybSDK_VERSION  - The version of the CybSDK  which was found
-#
-#
+#   CybSDK_FOUND         - True if the system has the Sense Glove SDK
+#   CybSDK_INCLUDE_DIR  - The include directory of the SDK
+#   CybSDK_LIBRARIES     - The name of the libraries to link against.
+###############################
```

Comment:
In modern CMake, it is better to define targets. Recently, I wrote these lines for a similar purpose (need to be tested)
```cmake
    if (BUILD_SHARED_LIBS)
        set(EXTENSION ${CMAKE_SHARED_LIBRARY_SUFFIX})
        set(TYPE "SHARED")
    else()
        set(EXTENSION ${CMAKE_STATIC_LIBRARY_SUFFIX})
        set(TYPE "STATIC")
    endif()

    if(${CMAKE_SIZEOF_VOID_P} EQUAL 8)
        # 64 bits
        set(FOLDER "x64")
    elseif(${CMAKE_SIZEOF_VOID_P} EQUAL 4)
        # 32 bits
        set(FOLDER "x86")
    endif()

    add_library(CybSDK ${TYPE} IMPORTED GLOBAL ${CybSDK_DIR}/${FOLDER}/CybSDK${EXTENSION})
    set_target_properties(CybSDK PROPERTIES IMPORTED_LOCATION ${CybSDK_DIR}/${FOLDER}/CybSDK${EXTENSION})
    target_include_directories(CybSDK INTERFACE ${CybSDK_DIR}/include)
```

The first two ``if``s are only needed to determine the type of library to link. The last three lines are the important ones. The first one generates the imported target, the second embeds in the target the location of the library, the third define from where to take the ``.h`` files. Then, you can simply use this target in other components by doing
```cmake
target_link_libraries(targetName CybSDK)
```
without changing anything else.